src/sentry/static/sentry/app/views/settings/incidentRules/ruleForm/index.tsx

```diff
@@ -331,11 +331,17 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
         onSubmitSuccess(resp, model);
       }
     } catch (err) {
-      const apiErrors = Array.isArray(err?.responseJSON)
-        ? `: ${err.responseJSON.join(', ')}`
-        : err?.responseJSON?.nonFieldErrors
-        ? `: ${err.responseJSON.nonFieldErrors.join(', ')}`
-        : '';
+      let errors: string[] = [];
+      if (Array.isArray(err?.responseJSON)) {
+        errors = err.responseJSON;
```

Comment:
If `responseJSON` is an array, are we guaranteed that it'll be an array of strings?